requirements.txt

```diff
@@ -1,3 +1,3 @@
-setuptools
+setuptools>=18.0
```

Comment:
@tjtg You would think that we don't need to raise the minimum version of `setuptools` here, as this `requirements.txt` is only used by `travis-ci` to install via `conda` and not `pip`... But if we changed our choice of installer, then there is the possibility that everything would break again during install for `travis-ci`...

So let's also bump this up to the minimal modern version of `40.8.0` :wink:

Reply:
I've added a commit with this change.